pkg/cmd/pipeline/start.go

```diff
@@ -0,0 +1,174 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	errNoPipeline      = errors.New("missing pipeline name")
+	errInvalidPipeline = errors.New("invalid pipeline name")
+	errInvalidResource = errors.New("invalid resource parameter: ")
+	errInvalidParam    = errors.New("invalid param parameter: ")
+)
+
+type triggerOptions struct {
```

Comment:
it is better to follow the convention of `verbCommand` -> `verbOptions` e.g. `logCommand` -> `logOptions` and there for `startCommand` -> `startOptions`

Reply:
Changed to startOptions